### ISSUE TITLE
Squiggly heredoc "<<~" unsupported

### ISSUE THREAD

**Issue by phiggins** [seattlerb/ruby_parser#218]
https://github.com/ruby/ruby/commit/9a28a29b870b5f45d370bc8f16c431b435f0bbb3

**Comment by tobiashm** [comment]
Parsing the following works (with 3.8.2)
```ruby
puts <<~eos
  Hello, World!
eos
```
But  if you use parentheses, it fails
```ruby
puts(<<~eos)
  Hello, World!
eos
```

**Comment by eoinkelly** [comment]
Parsing the following fails in 3.8.2
```
# ./do_thing.rb
def do_thing
  <<~EOM
  Some content
  EOM
end
```

with the error

```
[2] pry(main)> RubyParser.new.parse File.read("./do_thing.rb")
Racc::ParseError: (string):2 :: parse error on value "<<" (tLSHFT)
from /Users/eoinkelly/.rbenv/versions/2.3.1/lib/ruby/2.3.0/racc/parser.rb:528:in `on_error'
```

**Comment by phiggins** [comment]
This was fixed in #243 and has been released in the newest version.
